app/views/spree/admin/orders/_invoice_table.html.haml

```diff
@@ -2,13 +2,15 @@
   %thead
     %tr
       %th{:align => "left"}
-        %h4= t(:invoice_column_item)
+        %h5= t(:invoice_column_item)
       %th{:align => "right", :width => "15%"}
-        %h4= t(:invoice_column_qty)
+        %h5= t(:invoice_column_qty)
       %th{:align => "right", :width => "15%"}
-        %h4= @order.total_tax > 0 ? t(:invoice_column_tax) : ""
+        %h5= @order.has_taxes_included ? t(:invoice_column_unit_price_with_taxes) : t(:invoice_column_unit_price_without_taxes)
       %th{:align => "right", :width => "15%"}
-        %h4= t(:invoice_column_price)
+        %h5= @order.has_taxes_included ? t(:invoice_column_price_with_taxes) : t(:invoice_column_price_without_taxes)
+      %th{:align => "right", :width => "15%"}
+        %h5= @order.total_tax > 0 ? t(:invoice_column_tax_rate) : ""
   %tbody
```

Comment:
If we do this, we can just hide the tax column altogether for orders without any tax (common in Australia):

```ruby
- if @order.total_tax > 0
  %th{:align => "right", :width => "15%"}
    %h5= t(:invoice_column_tax_rate)
```

Obviously need to do the same thing for the column itself below...